nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java

```diff
@@ -112,8 +112,9 @@
     protected boolean active = true;
 
     @Override
-    protected String getName() {
-        return NAME;
+    public void setName(String name) {
+        // we hardcode a name so that subclasses (StreamWorkManager) use the same registry
+        super.setName(NAME);
     }
```

Comment:
it's weird, wouldn't it be better if the setter was disabled and the super.setName called in constructor ?

Reply:
It's roughly equivalent but ok it may make the intent clearer.